modules/virt-inferfromvolume-labels.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-specifying-instance-preference.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="inferfromvolume-labels_{context}"]
+= Setting the `inferFromVolume` labels
+
+Use the following labels on your PVC, data source, or data volume to instruct the inference mechanism which instance type, preference, or both to use when trying to boot from a volume.
+
+* `instancetype.kubevirt.io/default-instancetype`
+* `instancetype.kubevirt.io/default-instancetype-kind`
+** Optional. Otherwise, defaults to `VirtualMachineClusterInstancetype`.
+* `instancetype.kubevirt.io/default-preference`
+* `instancetype.kubevirt.io/default-preference-kind`
+** Optional. Otherwise, defaults to `VirtualMachineClusterPreference`.
+
+.Prerequisites
```

Comment:
@0xFelix Do we have any prereqs for this?

Reply:
No, I don't think so. Other than that the instancetype and preference need to exist on the cluster.